Customizing/global/plugins/Services/Repository/RepositoryObject/ReportEmplEduBios/classes/class.ilObjReportEmplEduBios.php

```diff
@@ -13,8 +13,18 @@ public function initType() {
 
 	public function __construct($ref_id = 0) {
 		parent::__construct($ref_id);
+		global $lng;
+		$this->gLng = $lng;
 	}
-	
+
+	// This is a super evil hack to make #2262 happen for this report.
+	// If there is a correct solution, we also could re-finalize getDescription
+	// in ilObject2.
+	public function getDescription() {
+		$this->gLng->loadLanguageModule("gev");
+		return $this->gLng->txt("gev_rep_attendance_by_employee_desc_short");
+	}
+
```

Comment:
Ich glaube das brauchen wir nicht mehr, oder? Als Beschreibung kann doch nun die in den Einstellungen angegebene Beschreibung verwendet werden, weil der Hilfetext im Tooltip ist. Oder checke ich was nicht?